frontend_lib/webpack.config.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
   mode: isProduction ? 'production' : 'development',
   entry: {
     lib: isProduction ? './src/index.js' : './src/index.dev.js',
+    testUtils: './test/index.js',
```

Comment:
the file name will be `tracim_frontend_lib.testUtils.js`, it will mix snake case and camel case which isn't great.
Prefer `test_utils`